int64 step_size
int64 bias_param
int64 max_iteration
